src/server/typingsInstaller/nodeTypingsInstaller.ts

```diff
@@ -85,6 +85,11 @@ namespace ts.server.typingsInstaller {
                 throttleLimit,
                 log);
             this.npmPath = npmLocation !== undefined ? npmLocation : getDefaultNPMLocation(process.argv[0]);
+
+            // If the NPM path contains spaces and isn't wrapped in quotes, do so.
+            if (this.npmPath.indexOf(" ") !== -1 && this.npmPath[0] !== `"`) {

```

Comment:
Just for clarity, could you do this only for `npmLocation` and not for `getDefaultNpmLocation` where it's presumably irrelevant?